Makefile

```diff
@@ -308,20 +308,20 @@ test-valgrind: all
 test-check-deopts: all
 	$(PYTHON) tools/test.py $(PARALLEL_ARGS) --mode=$(BUILDTYPE_LOWER) --check-deopts parallel sequential
 
-benchmark/napi/function_call/build/Release/binding.node: all \
+benchmark/napi/function_call/build/Release/binding.node: \
 		benchmark/napi/function_call/napi_binding.c \
 		benchmark/napi/function_call/binding.cc \
-		benchmark/napi/function_call/binding.gyp
-	$(NODE) deps/npm/node_modules/node-gyp/bin/node-gyp rebuild \
+		benchmark/napi/function_call/binding.gyp | all
+	@$(NODE) deps/npm/node_modules/node-gyp/bin/node-gyp rebuild \
```

Comment:
As a general rule I prefer not to silence make commands without an escape hatch via V=1

Reply:
Maybe if we make a rule for `binding.gyp` files we could catch two birds. (1) get it to echo iff V=1 and DRY the code.

Reply:
Something like:
```Makefile

quiet_cmd_node_gyp = NODE-GYP($(TOOLSET)) $@
cmd_node_gyp = $(NODE) deps/npm/node_modules/node-gyp/bin/node-gyp rebuild --python="$(PYTHON)" --nodedir="$(shell pwd)" --directory="$(abspath $(dir $(dir $(dir $@))))"

$(obj).$(TOOLSET)/binding.node: $(obj).$(TOOLSET)/binding.gyp FORCE_DO_CMD
	@$(call do_cmd,node_gyp,1)
```

Reply:
This PR was not intended to be about silencing the command, but not having this recipe executed every time. I added the `@` by mistake. I'll update shortly, thanks.